### PR TITLE
JDK-8297007: IGV: Link/Unlink node selection of open tabs

### PR DESCRIPTION
**Pull request by tobiasholenstein**
In IGV graphs can be opened in several tabs and then display them side-by-side. Previously, when the user selected nodes in tab A the selection was also applied in tab B. 

We now introduce a new global button to link and unlink the selection of different tabs.
link_button

If the button is **pressed**, the selection is **linked** globally across tabs: 
linked

If the button is **not pressed**, the selection is **not linked** across tabs. This is the default setting:
unlinked

# Implementation
The `SelectionCoordinator` is responsible to update the other tabs when the selection changes. We simply disable the `SelectionCoordinator` when the link button is not pressed, and enable it otherwise.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8297007](https://bugs.openjdk.org/browse/JDK-8297007): IGV: Link/Unlink node selection of open tabs

### Reviewers
 * [Roberto Castañeda Lozano](https://openjdk.org/census#rcastanedalo) (@robcasloz - **Reviewer**) ⚠️ Review applies to [4accab3e](https://git.openjdk.org/jdk/pull/11171/files/4accab3ed5c9d0094bfecc9912aa9b50671a9209)
 * [Christian Hagedorn](https://openjdk.org/census#chagedorn) (@chhagedorn - **Reviewer**) ⚠️ Review applies to [4accab3e](https://git.openjdk.org/jdk/pull/11171/files/4accab3ed5c9d0094bfecc9912aa9b50671a9209)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11171/head:pull/11171` \
`$ git checkout pull/11171`

Update a local copy of the PR: \
`$ git checkout pull/11171` \
`$ git pull https://git.openjdk.org/jdk pull/11171/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11171`

View PR using the GUI difftool: \
`$ git pr show -t 11171`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11171.diff">https://git.openjdk.org/jdk/pull/11171.diff</a>

</details>

### CODE REVIEW

File: src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/actions/GlobalSelectionAction.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
```
Comment (robcasloz) [inline src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/actions/GlobalSelectionAction.java:2]: Set new copyright year.

File: src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/actions/GlobalSelectionAction.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ */
+package com.sun.hotspot.igv.view.actions;
+
+import com.sun.hotspot.igv.view.EditorTopComponent;
+import java.awt.event.ActionEvent;
+import javax.swing.AbstractAction;
+import javax.swing.Action;
+import javax.swing.ImageIcon;
+import org.openide.util.ImageUtilities;
+
```
Comment (robcasloz) [inline src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/actions/GlobalSelectionAction.java:32]: Would be nice to assign a shortcut to this action.

**Review by robcasloz — APPROVED** [review]
Great improvement, this will really enable side-by-side viewing/exploration.

File: src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/actions/GlobalSelectionAction.java (hunk 1 above)
Comment (tobiasholenstein) [reply src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/actions/GlobalSelectionAction.java]: done

**Review by robcasloz — APPROVED** [review]
Thanks for addressing my comments, Tobias!

**Review by chhagedorn — APPROVED** [review]
Looks good! Works as expected on Linux.

**Pull request closed without merging**